src/Mvc/Mvc.Core/src/Formatters/ResponseContentTypeHelper.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) .NET Foundation. All rights reserved.
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
+#nullable enable
```

Comment:
Remove?

Reply:
This is shared with the Json.NET implementation that isn't currently nullable enabled.